install

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+success() { printf " \033[1;32m ✔ %s\033[0m\n" "$*"; }
```

Comment:
Pretty printing/formatting not needed for this script.
We should give a clear error message and exit 1 or 0 based on the final outcome